plugins/woocommerce/includes/admin/class-wc-admin-notices.php

```diff
@@ -252,6 +252,24 @@ public static function add_custom_notice( $name, $notice_html ) {
 		update_option( 'woocommerce_admin_notice_' . $name, wp_kses_post( $notice_html ) );
 	}
 
+	/**
+	 * Render a custom contextual notice.
+	 *
+	 * This is a wrapper for a contextual notice template.
+	 * Allows Woo or extensions to render a relevant notice in
+	 * a specific section of the Woo UI.
+	 * For example, rendering a payment dispute notice in order
+	 * edit screen, using `woocommerce_order_details_after_payment_info` hook.
+	 *
+	 * @param string $notice_html Notice message. Can be rich html (e.g. bold, italics, links).
+	 * @param string $severity    Severity of the notice. Can be 'error', 'warning', 'success', 'info'.
+	 * @param string $button_text Button text (plain text).
+	 * @param string $button_url  URL to link to.
+	 */
+	public static function render_contextual_notice( $notice_html, $button_text, $severity, $button_url ) {
```

Comment:
It looks like `$severity` is supposed to come before `$button_text` in the parameter order?

Reply:
Good spotting! Fixed here 4c9df162ce428997f54d3356c49ab991d0329b52